include/picongpu/param/incidentField.param

```diff
@@ -143,19 +143,29 @@ namespace picongpu
             using ZMax = profiles::None;
             /**@}*/
 
-            /** Inward offset of the Huygens surface from global domain boundaries
+            /** Position in cells of the Huygens surface relative to start of the total domain
              *
-             * The offset is in cells, counted from the corrresponding boundary inwards in the normal direction.
-             * This array is logically similar to specifying absorber thickness.
-             * The offset for the boundary must be at least absorber_thickness + (FDTD_spatial_order / 2 - 1).
-             * However beware of setting offset = absorber_thickness + const, as then changing absorber parameters
-             * may also affect laser positioning.
-             * When all used profiles are None, the check for OFFSET validity is skipped.
+             * The position is set as an offset, in cells, counted from the start of the total domain.
+             * For the max boundaries, negative position values are allowed.
+             * These negative values are treated as position at (global_domain_size[d] + POSITION[d][1]).
+             * It is also possible to specify the position explicitly as a positive number.
+             * Then it is on a user to make sure the position is correctly calculated wrt the grid size.
+             *
+             * Except moving window simulations, the position must be inside the global domain.
+             * The distance between the Huygens surface and each global domain boundary must be at least
+             * absorber_thickness + (FDTD_spatial_order / 2 - 1). However beware of setting position = direction *
+             * (absorber_thickness + const), as then changing absorber parameters will affect laser positioning. When
+             * all used profiles are None, the check for POSITION validity is skipped.
+             *
+             * For moving window simulations, POSITION[1][1] can be located outside the initially simulated volume.

```

Comment:
PLease extent the comment that `POSITION[1][1] ` positive side in Y direction to help the reader because `[1][1]` is hard to understand if you are not familiar with C array syntax.